website/docs/r/managed_cluster.html.markdown

```diff
@@ -0,0 +1,110 @@
+---
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_managed_cluster"
+sidebar_current: "docs-azurerm-resource-managed-cluster"
+description: |-
+  Creates an AKS Managed Cluster instance.
+---
+
+# azurerm\_managed\_cluster
```

Comment:
_minor:_ we don't actually need these backslashes

Reply:
Thanks, fixed.